doc/tutorials/views/index.rst

```diff
@@ -2,7 +2,7 @@
 
 .. _view-tutorial:
 
-Using Views for Start/End Screens
+Basic Non-Game Screens with Views
```

Comment:
I get the generic wording is more accurate, but in my experience teaching, students tended to ask for start/end screens most often. When a student wants a "game over" screen, I'm not sure she'd know to look up "Non-game screens with views." I prefer the original in this case as well.

Reply:
reverted :+1: